apps/aqua/src/index.ts

```diff
@@ -122,6 +159,133 @@ app.post("/logout", (c) => {
   return c.redirect("/");
 });
 
+app.get("/stamp", (c) => {
+  // check logged in
+  const tealSession = getCookie(c, "tealSession");
+  if (!tealSession) {
+    return c.redirect("/login");
+  }
+  return c.html(
+    `
+    <head>
+    <link rel="stylesheet" href="https://latex.vercel.app/style.css">
+    </head>
+    <div id="root">
+    <div id="header">
+      <h1>teal.fm</h1>
+      <p>Your music, beautifully tracked. (soon.)</p>
+      <div style=" width: 100%; display: flex; flex-direction: row; justify-content: space-between; gap: 0.5rem;">
+        <div>
+          <a href="/">home</a>
+          <a href="/stamp">stamp</a>
+        </div>
+        <form action="/logout" method="post" class="session-form">
+          <button type="submit" style="background-color: #cc0000; color: white; border: none; padding: 0rem 0.5rem; border-radius: 0.5rem;">logout</button>
+        </form>
+      </div>
+    </div>
+    <div class="container">
+      <p>want to share what you're listening to in the meantime??</p>
```

Comment:
imo, maybe we should explain that this is not the full product, like yeah "in the meantime" kinda gets that point across, but most will probably skip past that

something like:
> while we are building out our music tracker, try out this!

idk